OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+will contain 2 arrays of equal length where the index of each entry
+corresponds to the same sample. For example, if a sample had the
+SampleID of `Sample_1_Aliquot_1` in the first project and
+`Sample_1_Aliquot_2` in the second project, then the overlap sample list
+should look as the following.
+
+```{r eval=FALSE, include=TRUE}
+overlap_sample_list <-list("DF1" = c("A1", "A2", "A3", "Sample_1_Aliquot_1"),
+                           "DF2" = c("A1", "A2", "A3",  "Sample_1_Aliquot_2")) 
+```
+
+Note that external controls should not be included in the list of
+bridging samples, as detailed in the [Bridge Sample Selection] section
+of this tutorial. External control samples often share the same naming
+convention across data sets but may represent different samples due to
+reagent batch differences. Appending the project name to the end of the
+control samples can ensure unique Sample IDs.
+
+```{r, echo=TRUE, eval=FALSE}
+data.frame(SampleID = intersect(data_E3072$SampleID, data_EHT$SampleID)) |>
+  dplyr::filter(!stringr::str_detect(SampleID, "CONTROL_SAMPLE"))
+```
+
+```{r, echo=FALSE, eval=FALSE}
+data.frame(SampleID = intersect(data_E3072$SampleID, data_EHT$SampleID)) |>
+  dplyr::filter(!stringr::str_detect(SampleID, "CONTROL_SAMPLE")) |> #Remove control samples
+  kableExtra::kbl(booktabs = TRUE,
+      digits = 2,
+      caption = "Table 3. Overlapping bridging samples") |>
+  kableExtra::kable_styling(bootstrap_options = "striped", full_width = FALSE,
+                position = "center", latex_options = "HOLD_position")
+```
+
+```{r, message=FALSE, fig.cap=f3, eval=FALSE}
+#### Extract bridging samples
+
+overlapping_samples <-  data.frame(SampleID = intersect(data_E3072$SampleID, data_EHT$SampleID)) |>  
+  filter(!str_detect(SampleID, "CONTROL_SAMPLE")) |> #Remove control samples
+  pull(SampleID)
+
+npx_before_br <- data_E3072 |>
+  dplyr::filter(!str_detect(SampleID, "CONTROL_SAMPLE")) |> #Remove control samples
+  dplyr::mutate(Type = if_else(SampleID %in% overlapping_samples,
+                        paste0("Explore 3072 Bridge"),
+                        paste0("Explore 3072 Sample"))) |>
+  rbind({
+    data_EHT |>
+      filter(!str_detect(SampleID, "CONTROL_SAMPLE")) |> #Remove control samples |> 
+      mutate(Type = if_else(SampleID %in% overlapping_samples,
+                            paste0("Explore HT Bridge"),
+                            paste0("Explore HT Sample"))) |>
+      mutate(SampleID = if_else(SampleID %in% overlapping_samples,
+                                paste0(SampleID, "_new"),
+                                SampleID))
+  })
+
+### PCA plot
+OlinkAnalyze::olink_pca_plot(df          = npx_before_br,
+                             color_g     = "Type")
+```
+
+## Normalization
+
+The `olink_normalization_bridge_product()` function is used to determine
+which assays are bridgeable and of the bridgeable assays what
+normalization method is advised, and to calculate normalized NPX values
```

Comment:
Might be good to split this for easier readability - possibly something like the following:

> The `olink_normalization_bridge_product()` function is used to determine which assays are bridgeable, and of the bridgeable assays, which normalization method is advised. This function also calculates normalized NPX values
> for the Explore 3072 (non-reference) project.

Reply:
Also TBD - will we have the olink_normalization_bridge_product() wrapper ready for initial launch? If not, should we just refer to olink_normalization()? Will leave as-is for now.